source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
Thanks for the information.

@tkoeppe: Ok to redefine \tcode to the second form (allowing line breaks)? (In non-math-mode, we definitely want linebreaks inside \tcode.)